prow/config/config.go

```diff
@@ -1125,6 +1151,14 @@ func (d *Deck) Validate() error {
 		}
 	}
 
+	if d.DefaultRerunAuthConfigs != nil {
+		for k, config := range d.DefaultRerunAuthConfigs {
+			if err := config.Config.Validate(); err != nil {
+				return fmt.Errorf("default_rerun_auth_configs[%d]: %w", k, err)
+			}
+		}
+	}
```

Comment:
Go is smart on working with nil slice, so this could be:

```suggestion
		for k, config := range d.DefaultRerunAuthConfigs {
			if err := config.Config.Validate(); err != nil {
				return fmt.Errorf("default_rerun_auth_configs[%d]: %w", k, err)
			}
		}
```

Reply:
Thanks for the catch. fixed!